### PR TITLE
Add a way to measure test coverage

### PR DESCRIPTION
**Pull request by tadzik**
We had some leftovers of it in .gitignore, now it's actually available as a command

### CODE REVIEW

**Review by Cadair — APPROVED** [review]
I wont pretend to be able to fully understand this incantation, but if it works then horray.

File: package.json [hunk 1]
```diff
@@ -85,6 +86,8 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "js-yaml": "^4.1.0",
     "mocha": "^10.0.0",
+    "node-mocks-http": "^1.14.1",
```
Comment (Half-Shot) [inline package.json]: hello?

**Review by Half-Shot — CHANGES_REQUESTED** [review]
Changelog please :)

File: package.json (hunk 1 above)
Comment (tadzik) [reply package.json]: [Is it me you're looking for?](https://youtu.be/Cwnrsd5so9M?t=27)

(Removed in dc9bc72b146c931b6447f386b8aa844e5ebec073, thanks!)

**Pull request merged**